docs/docs/install/docker-compose/index.md

```diff
@@ -44,7 +48,145 @@
 !!! info "Getting help"
     If you run into any problems during installation, see [troubleshooting](./troubleshooting.md) or [open a ticket describing your problem](https://github.com/centerofci/mathesar/issues/new/choose).
 
-## Start/stop the server {:#start-stop}
+
+## Manual Install
+
+1. Open the Mathesar configuration directory where you'd like to install Mathesar. By convention, we do it within `/etc/mathesar`
+
+2. Download the necessary files into the configuration directory
+
+    - Download the `docker-compose.yml` file from [https://github.com/centerofci/mathesar/raw/master/docker-compose.yml](https://github.com/centerofci/mathesar/raw/master/docker-compose.yml)
+     - Download the example `.env.example` file used for setting the [configuration variables](../configuration.md) from [https://github.com/centerofci/mathesar/raw/master/.env.example](https://github.com/centerofci/mathesar/raw/master/.env.example)
+     - Rename the downloaded `.env.example` to `.env`
+     - Download the `Caddyfile` from [https://github.com/centerofci/mathesar/raw/master/Caddyfile](https://github.com/centerofci/mathesar/raw/master/Caddyfile).
```

Comment:
This step is not needed as our prebuilt docker image already has the Caddyfile

Reply:
I addressed this in d2d23e441adf85604ace7d76ce01839301d5ddb7

Reply:
Thanks!